src/tests/ftest/cart/no_pmix/cart_nopmix_multictx_one_node.py

```diff
@@ -52,11 +44,21 @@ def test_cart_no_pmix(self):
         :avocado: tags=all,cart,pr,daily_regression,no_pmix,one_node
         """
 
+        crt_phy_addr = os.environ.get("CRT_PHY_ADDR_STR")
```

Comment:
i wonder if we even have to do this now, as we already set env fully. we might revisit later and have things cleaned up so that we dont keep cloning envs if the base env is sufficient already